zerver/lib/users.py

```diff
@@ -50,7 +50,7 @@
 )
 
 
-def check_full_name(full_name_raw: str) -> str:
+def check_full_name(full_name_raw: str, realm: Realm) -> str:
```

Comment:
Should this be declared as `Optional[Realm`]? I think `clean_full_name`, at least, will pass `None` here.

(We don't want to declare a default; this should be a required parameter).

Reply:
updated.